website/markdown/posts/2020-12-24-next-for-tuist/post.mdx

```diff
@@ -0,0 +1,78 @@
+
+Since Tuist knows about the project dependency tree and the file changes by using the underlying a version control tool like Git, it can combine that information with some other heuristics to provide selective building and testing and speed up developers' workflows.
+
+By letting Tuist optimize the workflows, teams will automatically get optimizations introduced in future releases. Furthermore, they can focus their efforts on building the apps and not defining an efficient CI workflow.
+
+## Caching improvements
+
+In 2020 we introduced the concept of generating **focused projects**. Focused projects are projects optimized to work on a given target. For example, if I plan to work on app `MyApp` and the framework `MyAppSupport`, I can run `tuist focus MyApp MyAppSupport`, and Tuist will remove the targets that are not necessary to work on those targets, and replace direct and transitive dependencies with a precompiled versions of them. The latter is something that we refer to as **caching**.
+
+Unlike modern build systems like [Buck](https://buck.build) and [Bazel](https://bazel.build) that cache at the file level at the cost of taking developers away from Xcode's build system and having to keep up with Xcode updates, Tuist's caching works at the module level and using Xcode's build system. It provides a command, `tuist cache warm`, to precompile and cache every project's cacheable target.
+
+Caching works great with some ideal project scenarios. However, there are many complex project scenarios that we don't handle gracefully. For example, the caching of projects with transitive Swift Packages as dependencies doesn't work. It's not impossible, but it'll certainly require a fair amount of work. We are aligning our work on better management of third-party dependencies to allow their caching. By the end of 2021, we hope to cache most of the target types: _frameworks, libraries, bundles, and third-party dependencies._
+
+## App releasing
+
+Releasing an app involves many steps: bumping the app version, archiving the app, exporting it with the right signing profiles, and uploading it to Testflight. Tuist could streamline all of them into a single workflow that gets triggered by running:
+
+```
+tuist release
+```
+
+The command would work both locally and on CI, signing the app with the right provisioning profile and certificate, and authenticating and uploading the app through the [App Store Connect API](https://developer.apple.com/app-store-connect/api/).
+
+## Tuist 2.0
+
+Tuist's API has been very stable for the last years, and we have a good amount of features to justify the release of a new major version. Therefore, 2021 will be the year when we'll release a new major version of Tuist, 2.0. To make this new release special, we are working on a **redesign of our brand and website**. We are taking the opportunity to revisit the website's content to ensure the ideas are clearly conveyed and that users can find what they are looking for easily.
```

Comment:
for the last years -> for the last few years / two years would maybe be more suitable? (I don't think the API has been stable for more than two years 😅 )